controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -594,84 +581,6 @@ func getFailureDomainKeyFromStorageClassParameter(sc *storagev1.StorageClass) st
 	return util.GetFullTopologyLabel(failuredomain)
 }
 
-func (r *OCSInitializationReconciler) reconcileUXBackendSecret(initialData *ocsv1.OCSInitialization) error {
-
-	var err error
-
-	secret := &corev1.Secret{}
-	secret.Name = "ux-backend-proxy"
-	secret.Namespace = initialData.Namespace
-
-	_, err = ctrl.CreateOrUpdate(r.ctx, r.Client, secret, func() error {
-
-		if err := ctrl.SetControllerReference(initialData, secret, r.Scheme); err != nil {
-			return err
-		}
-
-		secret.StringData = map[string]string{
-			"session_secret": random30CharacterString,
-		}
-
-		return nil
-	})
-
-	if err != nil {
-		r.Log.Error(err, "Failed to create/update ux-backend secret")
-		return err
-	}
-
-	r.Log.Info("Secret creation succeeded", "Name", secret.Name)
-
-	return nil
-}
-
-func (r *OCSInitializationReconciler) reconcileUXBackendService(initialData *ocsv1.OCSInitialization) error {
-
-	var err error
-
-	service := &corev1.Service{}
-	service.Name = "ux-backend-proxy"
-	service.Namespace = initialData.Namespace
-
-	_, err = ctrl.CreateOrUpdate(r.ctx, r.Client, service, func() error {
-
-		if err := ctrl.SetControllerReference(initialData, service, r.Scheme); err != nil {
-			return err
-		}
-
-		service.Annotations = map[string]string{
```

Comment:
we might also be having the service in bundle manifest, possible to cross check once?